command/search_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright (C) 2018 Expedia Group.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"github.com/HotelsDotCom/flyte-jira/client"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestSearchIssuesSuccess(t *testing.T) {
+	client.SendRequest = func(request *http.Request, responseBody interface{}) (int, error) {
+		return http.StatusOK, nil
+	}
+
+	var inputStruct = struct {
+		Query string `json:"query"`
+	}{"project = FLYTE"}
+	input := toJson(inputStruct, t)
+
+	actualEvent := searchIssuesHandler(input)
```

Comment:
I think we can simplify it to:
```
actualEvent := searchIssuesHandler([]byte(`{"query": "project = FLYTE"}`))
```

Reply:
I've refactored it in b5302b9. It's indeed cleaner like this.